code/game/objects/items/stacks/medical.dm

```diff
@@ -149,3 +163,82 @@
 /obj/item/stack/medical/ointment/suicide_act(mob/living/user)
 	user.visible_message("<span class='suicide'>[user] is squeezing \the [src] into [user.p_their()] mouth! [user.p_do(TRUE)]n't [user.p_they()] know that stuff is toxic?</span>")
 	return TOXLOSS
+
+/obj/item/stack/medical/suture
+	name = "suture"
+	desc = "Sterile sutures used to seal up cuts and lacerations."
+	gender = PLURAL
+	singular_name = "suture"
+	icon_state = "suture"
+	self_delay = 30
+	other_delay = 10
+	amount = 15
+	max_amount = 15
+	repeating = TRUE
+	var/heal_brute = 10
+	grind_results = list(/datum/reagent/medicine/spaceacillin = 2)
+
+/obj/item/stack/medical/suture/medicated
+	name = "medicated suture"
+	icon_state = "suture_purp"
+	desc = "A suture infused with drugs that speed up wound healing of the treated laceration."
+	heal_brute = 15
+	grind_results = list(/datum/reagent/medicine/polypyr = 2)
+
+/obj/item/stack/medical/suture/heal(mob/living/M, mob/user)
+	. = ..()
+	if(M.stat == DEAD)
+		to_chat(user, "<span class='warning'>[M] is dead! You can not help [M.p_them()].</span>")
+		return
+	if(iscarbon(M))
+		return heal_carbon(M, user, heal_brute, 0)
+	to_chat(user, "<span class='warning'>You can't heal [M] with the \the [src]!</span>")
+
+/obj/item/stack/medical/mesh
+	name = "regenerative mesh"
+	desc = "A bacteriostatic mesh used to dress burns."
+	gender = PLURAL
+	singular_name = "regenerative mesh"
+	icon_state = "regen_mesh"
+	self_delay = 30
+	other_delay = 10
+	amount = 15
+	max_amount = 15
+	repeating = TRUE
+	var/heal_burn = 10
+	var/is_open = FALSE ///This var determines if the sterile packaging of the mesh has been opened.
+
+/obj/item/stack/medical/mesh/update_icon()
+	if(!is_open)
+		icon_state = "regen_mesh_closed"
+		return
+	. = ..()
+
+/obj/item/stack/medical/mesh/heal(mob/living/M, mob/user)
+	. = ..()
+	if(!is_open)
+		to_chat(user, "<span class='warning'>You need to open [src] first.</span>")
+		return

```

Comment:
This should probably be in try_heal instead of heal.

Reply:
Yeah, i will move it.